projects/tsq-gallery/src/lib/components/main-tsq-gallery/main-tsq-gallery.component.ts

```diff
@@ -72,12 +74,39 @@ export class TSqGalleryComponent {
   /** Option to change the displayed image when the files is of an unsupported format.  */
   @Input() invalidFormatDisplayImage: string;
 
+  /** Show the viewer inline instead of a dialog. */
+  @Input() displayInline = false;
+
+  /** When inline, always keeps the viewer open. */
+  @Input() keepOpen = false;
+
+  private componentDestroyed$ = new Subject();
+
+  ngAfterViewInit() {
+    if (this.displayInline && this.keepOpen) {
+      of(this.files)
```

Comment:
why this of?

Reply:
to transform files into an observable as they can change and thde lib needs to update accordly